laterpay/signing.py

```diff
@@ -62,33 +79,53 @@ def normalise_param_structure(params):
     """
     Canonicalise representation of key-value data with non-unique keys.
 
-    Request parameter dictionaries are handled in different ways in different libraries,
-    this function is required to ensure we always have something of the format:
-
-       {  key:  [ value1, value2... ] }
+    Request parameter dictionaries are handled in different ways in different
+    libraries. This function is required to ensure we always have something of
+    the format::
+
+        {
+            'key1': ['value1', 'value2'],
+            'key2': ['value3'],
+        }
+
+    :param dict, list, tuple, furl.omdict1D.omdict params: The parameter
+        structure to normalize. Can be either a ``dict``, ``list``, ``tuple``
```

Comment:
Not sure if it's going to make a difference specially considering this function already existed before this PR, but the function has `normalise` on tis name and here it's written `normalize`.